raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -430,7 +431,8 @@ public RaftClientTestContext build() throws IOException {
             client.initialize(
                 staticVoterAddressMap,
                 quorumStateStore,
-                metrics
+                metrics,
+                new ExternalKRaftMetrics(null, null)
```

Comment:
Where are the test for this metrics like you did for the new metrics in `KafkaRaftMetrics`?